jwst/pipeline/calwebb_guider.cfg

```diff
@@ -0,0 +1,10 @@
+name = "FGSPipeline"
+class = "jwst.pipeline.FGSPipeline"
```

Comment:
I think we should try not to use the generic name "FGS" too much, because the FGS instrument can also be used in science mode. Because all of this code is specific to handling guide-mode data, I suggest more explicit names like "GuiderPipeline" and "jwst.pipeline.GuiderPipeline", which is also consistent with the "calwebb_guider" name.